### DISCUSSION TITLE
Using Cookies with RSS

### DISCUSSION THREAD

**Discussion by erayymz** [Flexget/Flexget#3683] [General]
Hello,

I exported the cookie from firefox and saved it to a file. My config is below. It seems like it is not working. What am I doing wrong?

When I manually import the same cookie to a private browser page, and use the same RSS link, it does work from the browser just fine.

Cookie File Contents:
---------------------
```
# Netscape HTTP Cookie File

filelist.io	FALSE	/	FALSE	2147483643	uid	xxx
filelist.io	FALSE	/	FALSE	2147483643	pass	xxx
```

Configuration:
--------------
```
task1-rb:
    cookies:
      file: /root/.flexget/cookies/cookie.txt
      type: mozilla
    rss: xxx
    regexp:
      accept:
        - ^(?=.*test).*$: {from: title}
    download: /var/download/
```

Error Message:
---------------
```
2023-02-11 12:58:28 CRITICAL rss          task1-rb Received content looks a bit like login page
2023-02-11 12:58:28 CRITICAL rss           task1-rb I have saved the invalid content to /root/.flexget/received/task1-rb.xml for you to view
2023-02-11 12:58:28 CRITICAL plugin        task1-rb Received invalid RSS content from task task1-rb
```

**Comment by BrutuZ** [comment]
I found the `cookies` plugin to be wildly inconsistent and after a lot of trouble dealing with wildcard domains I just gave up on it entirely.
For tasks that need cookies (like CF protected pages), I pass them with the `headers` plugin instead:
```yaml
headers:
  "Cookie": "cookie1_name=cookie1_value; cookie2_name=cookie2_value"
```
